Blip.HttpClient/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -60,10 +60,11 @@ public static IServiceCollection RegisterBlipExtensions(this IServiceCollection
         /// <param name="authKey"></param>
         /// <param name="documentList"></param>
         /// <param name="protocol">Defaults to Http to avoid breaking changes</param>
+        /// <param name="domain">Domain of request in blip. Default is 'msging.net'</param>
         public static IServiceCollection DefaultRegister(this IServiceCollection services, string authKey,
-                                                         List<Document> documentList = null, BlipProtocol protocol = BlipProtocol.Http)
+                                                         List<Document> documentList = null, BlipProtocol protocol = BlipProtocol.Http, string domain = null)
```

Comment:
why `null` instead of simply passing the current default domain?

Reply:
because this is checked inside lib